CHANGELOG.md

```diff
@@ -16,7 +16,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Fix get event endpoint with user auth key. [#746](https://github.com/rokwire/rokwire-building-blocks-api/issues/746)
+
 ### Added
+- Fix published capabilities and talents not showing without login. [#767](https://github.com/rokwire/rokwire-building-blocks-api/issues/767)
```

Comment:
It looks like when you resolved merge conflicts, new sections got added. We need to update the existing entries under the [Unreleased] section. Please let me know if you have any questions. Thanks.